content/en/docs/reference/glossary/security-context.md

```diff
@@ -4,14 +4,22 @@ id: security-context
 date: 2018-04-12
 full_link: /docs/tasks/configure-pod-container/security-context/
 short_description: >
-  The securityContext field defines privilege and access control settings for a Pod or Container, including the runtime UID and GID.
+  The securityContext field defines privilege and access control settings for a Pod or container, including the runtime UID and GID.
 
 aka: 
 tags:
 - security
 ---
- The securityContext field defines privilege and access control settings for a Pod or Container, including the runtime UID and GID.
+ The `securityContext` field defines privilege and access control settings for
+a {{< glossary_tooltip text="Pod" term_id="pod" >}} or
+{{< glossary_tooltip text="ontainer" term_id="container" >}},
+including the runtime UID and GID.
```

Comment:
line 15, spelling of tooltip text `container`

Reply:
Thanks @kbhawkey! Your eagle eyes remain keen.